src/main/java/com/bokudos/bokudosserver/external/stagebuilder/Tiles.java

```diff
@@ -0,0 +1,26 @@
+package com.bokudos.bokudosserver.external.stagebuilder;
+
+import java.util.Map;
+import java.util.TreeMap;
+
+/**
+ * This class uses a HashMap, therefore it is not threadsafe.
+ * A different Tiles instance should be used per game even if the tiles are built from the same stage.
+ */
+public class Tiles {
+
+    private Map<Integer, Map<Integer, Boolean>> data;
```

Comment:
Nested map o.0

Reply:
haha yeah, we can definitely update this. I have been thinking of better ways to store the tile data.